XTS/WS-C/dev/src/com/arjuna/webservices11/wscoor/sei/SoapFaultPortTypeImpl.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2006, JBoss Inc., and individual contributors as indicated
+ * by the @authors tag.  All rights reserved. 
+ * See the copyright.txt in the distribution for a full listing 
+ * of individual contributors.
+ * This copyrighted material is made available to anyone wishing to use,
+ * modify, copy, or redistribute it subject to the terms and conditions
+ * of the GNU Lesser General Public License, v. 2.1.
+ * This program is distributed in the hope that it will be useful, but WITHOUT A
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License,
+ * v.2.1 along with this distribution; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA  02110-1301, USA.
+ * 
+ * (C) 2005-2006,
+ * @author JBoss Inc.
+ */
+package com.arjuna.webservices11.wscoor.sei;
+
+import com.arjuna.webservices.logging.WSCLogger;
+import com.arjuna.webservices11.wsaddr.AddressingHelper;
+import com.arjuna.wsc11.AsynchronousRegistrationMapper;
+import javax.annotation.Resource;
+import javax.jws.HandlerChain;
+import org.xmlsoap.schemas.soap.envelope.Fault;
+
+import javax.jws.Oneway;
+import javax.jws.WebMethod;
+import javax.jws.WebParam;
+import javax.jws.WebService;
+import javax.jws.soap.SOAPBinding;
+import javax.xml.ws.Action;
+import javax.xml.ws.WebServiceContext;
+import javax.xml.ws.handler.MessageContext;
+import javax.xml.ws.soap.Addressing;
+import org.jboss.ws.api.addressing.MAP;
+
+
+@WebService(targetNamespace = "http://jbossts.jboss.org/xts/soapfault",
+        name = "SoapFaultPortType", 
+        serviceName = "SoapFaultService",
+        portName = "SoapFaultPortType")
+@SOAPBinding(parameterStyle = SOAPBinding.ParameterStyle.BARE)
+@HandlerChain(file="/ws-c_handlers.xml")
+@Addressing(required=true)
+public class SoapFaultPortTypeImpl {
+
+    @Resource private WebServiceContext webServiceCtx;
+    
+    /**
+     * 
+     * @param fault
```

Comment:
I know this is not a rule in the WS Narayana in general but I think having empty javadoc does not help the readable code. What do you think in removing the /** */ completely here?